packages/clay-badge/src/ClayBadge.soy

```diff
@@ -0,0 +1,69 @@
+{namespace ClayBadge}
+
+/**
+ * This renders a badge.
+ */
+{template .render}
+	{@param? closeable: bool}
+	{@param? href: string}
+	{@param? id: string}
+	{@param? label: html|string}
+	{@param? size: string}
+	{@param? spritemap: string}
+	{@param? style: string}
+
+	{let $classes kind="text"}
+		badge badge-pill
+		{sp}badge-{$style ?: 'default'}
+
+		{if $size}
+			{sp}badge-{$size}
+		{/if}
+	{/let}
+
+	{let $attributes kind="attributes"}
```

Comment:
This seems to only be necessary inside the `else` statement, right?

Reply:
Yes, do you think we should put it right in the`span` tag?